### PR TITLE
Add steps to add custom resolver to README

### PR DESCRIPTION
**Pull request by divyenduz**
Closes https://github.com/graphcool/graphql-boilerplate/issues/15

Please let me know if any changes are needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,33 @@ yarn playground
 - [`/src/index.ts`](https://github.com/graphcool/graphql-boilerplate/blob/master/src/index.ts) is the entry point of your application, pulling everything together and starting the [`graphql-yoga`](https://github.com/graphcool/graphql-yoga) Server.
 - [`/src/resolvers/`](https://github.com/graphcool/graphql-boilerplate/tree/master/src/resolvers) includes the actual business logic of your application. In GraphQL you implement [resolvers](http://graphql.org/learn/execution/) that *resolve*  a specific query being requested
 
+### Next Steps
+#### Adding a custom resolver
+Suppose we want to add a custom resolver to delete a post (example). We can take the following steps:-
```
Comment (marktani) [inline README.md]: I prefer

> Suppose we want to add a custom resolver to delete a post. We can take the following steps:

File: README.md [hunk 2]
```diff
@@ -69,6 +69,33 @@ yarn playground
...
+
+Add `delete` field to the Mutation type in `src/schema.graphql`
```
Comment (marktani) [inline README.md]: I prefer
> Add a new `delete` field to the Mutation type in `src/schema.graphql`

File: README.md [hunk 3]
```diff
@@ -69,6 +69,33 @@ yarn playground
...
+```
+type Mutation {
+  createDraft(title: String!, text: String): Post
+  publish(id: ID!): Post
+  delete(id: ID!): Post
+}
+```
+
+Add `delete` resolver to Mutation part of `src/index.js`
```
Comment (marktani) [inline README.md]: I prefer

> Add a `delete` resolver to Mutation part of `src/index.js`

File: README.md [hunk 4]
```diff
@@ -69,6 +69,33 @@ yarn playground
...
+  {
+    where: { id }
+  },
+    info
+  );
+}
+```
+
+Run `yarn start`
+
```
Comment (marktani) [inline README.md]: Add the example how to add a post:

```graphql
mutation {
  delete(id: "cuid") {
    id
  }
}
```

Comment (divyenduz) [reply README.md]: I suppose you mean, how to delete a post? Made the required changes. Let me know if anything else needed :)

**Comment by marktani** [comment]
Thanks 🙌

**Comment by marktani** [comment]
For now, I am collecting common scenarios in this issue instead: https://github.com/graphql-cli/graphql-boilerplate/issues/35. I moved your descriptions there, as well.

Once we collected a few and the `README`s of the individual boilerplates have been reorganized, we can add them back in where appropriate 🙂

**Comment by divyenduz** [comment]
Sounds good!

**Pull request merged**